modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/ScriptPathAttributeGenerator.cs

```diff
@@ -22,12 +22,15 @@ public void Execute(GeneratorExecutionContext context)
 
             // NOTE: NotNullWhen diagnostics don't work on projects targeting .NET Standard 2.0
             // ReSharper disable once ReplaceWithStringIsNullOrEmpty
-            if (!context.TryGetGlobalAnalyzerProperty("GodotProjectDir", out string? godotProjectDir)
+            if (!context.TryGetGlobalAnalyzerProperty("GodotProjectDirBase64", out string? godotProjectDir)
                 || godotProjectDir!.Length == 0)
             {
                 throw new InvalidOperationException("Property 'GodotProjectDir' is null or empty.");
```

Comment:
```suggestion
                throw new InvalidOperationException("Property 'GodotProjectDirBase64' is null or empty.");
```

Alternatively fallback to the old property for compatibility with existing projects that directly referencing the generator,

Reply:
Yes, even though `Godot.NET.Sdk` references the newer version, we should avoid breaking compat.